### DISCUSSION TITLE
Announcing dhcp-probe :tada:

### DISCUSSION THREAD

**Discussion by troglobit** [libnet/libnet#165] [Show and tell]
While working on the libnet v1.3 release I discovered APIs that referred to the [dhcp-probe](https://www.net.princeton.edu/software/dhcp_probe/) project.   I noticed maintenance had been discontinued by the dept. so I asked if we could adopt it to the libnet org.

On Mon, Oct 2, 4:26 PM, Irwin S. Tillman replied:
> Yes, you are welcome to adopt the dhcp-probe project into the libnet family of projects.

So allow me to present: <https://github.com/libnet/dhcp-probe> :smiley::tada:

**Comment by ivalery111** [comment]
I saw you had already started working on it:)
